### PR TITLE
Aggregation of partial results

### PR DESCRIPTION
**Pull request by schoersch**
see #152

### CODE REVIEW

**Comment by schoersch** [comment]
@smilesun I've added the new command line arg.
I think this is more flexible then inside the snakefile. Also, the arg allows to intermediate results of a still running benchmark if the state of the output dir can be downloaded to a machine.

**Comment by schoersch** [comment]
@smilesun Note that I'm not available next week. I don't know if you count this pr to the ones to be merged in the next days. If not, I can provide possible changes next weekend.

**Review by smilesun — CHANGES_REQUESTED** [review]
Traceback (most recent call last):
  File "/home/sunxd/DomainLab/main_out.py", line 8, in <module>
    if args.bm_config:
AttributeError: 'Namespace' object has no attribute 'bm_config'

**Comment by schoersch** [comment]
> Traceback (most recent call last): File "/home/sunxd/DomainLab/main_out.py", line 8, in if args.bm_config: AttributeError: 'Namespace' object has no attribute 'bm_config'

Thank you for pointing out, should be fixed now.

File: domainlab/utils/generate_benchmark_plots.py [hunk 1]
```diff
@@ -31,13 +32,16 @@ def gen_benchmark_plots(agg_results: str, output_dir: str, use_param_index: bool
 
     agg_results: path to the csv file
     output_dir: path to a folder which shall contain the results
+    skip_gen: Skips the actual plotting, used to speed up testing.
```
Comment (smilesun) [inline domainlab/utils/generate_benchmark_plots.py:35]: This should be a new feature for plots?

**Pull request merged**